socialmediapolicy.md

```diff
@@ -0,0 +1,155 @@
+# Social Media Policy v0.1.0
+
```

Comment:
Let's add in a "How to use" paragraph.
I am thinking on the lines of...

"This document is a ......., it is intended to be as light touch as possible allowing while still being safe online, this style would allow a school to use this policy as a base they could then lock down if needed."

While I know we may have that in another area of the repo I think that fact needs to be at the top of the document so people know they can lock down other stuff is needed.